### DISCUSSION TITLE
What databases and citation managers do you use most?

### DISCUSSION THREAD

**Discussion by ADParedes** [Open-Research-Program/HCOE#2] [Q&A]
What databases and citation managers do you use most?  Examples of reputable biomedical databases are Pubmed, SCOPUS, and Medline.  An example of a citation manager is Endnote and Mendeley

**Comment by Jdlopez0108** [comment]
I pretty much exclusively use Pubmed. I'd be interested to learn more about using other databases, specifically UpToDate, as I've had trouble gaining access to it. I'm pretty sure we're supposed to have access to UpToDate as medical students and I even think I've used it before, but in recent times I've had trouble gaining access.

**Reply by cjuare7** [reply]
https://library.uic.edu/libraries/lhs-chicago/
https://library.uic.edu/libraries/lhs-peoria/

Love UpToDate. I always found the easiest way to access it is through our schools library website. I like using the Peoria one because they have more links for the popular databases and easy access rather than looking through the A-Z databases
